terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/redirect/RedirectTest.java

```diff
@@ -105,12 +103,22 @@ public void test01_03_redirectToExternalLink() {
         // screen capture
         screenCapture.save(driver);
 
+        ApServerName apServerName = webDriverOperations.getApServerName();
+        String apServerVersion = webDriverOperations.getApServerVersion();
+
         driver.findElement(By.id("btn1")).click();
 
+        // Unlike other application servers, WebSphere Liberty Profile 16 & WebSphere traditional 9 wraps an unexpected exception of ServletException.
+        // So the expected error page is different.
         // confirms that 404 error occurred after login transition
-        assertThat(driver.findElement(By.xpath("/html/body/div/h2")).getText(),
-                is(pageTitle404Error));
-
+        if (apServerName == ApServerName.WEBSPHERELP && "16".equals(apServerVersion)
+                || apServerName == ApServerName.WEBSPHERETR && "9".equals(apServerVersion)) {

```

Comment:
Comment: `// Unlike other application servers, WebSphere Liberty Profile 16 & WebSphere traditional 9 wraps an unexpected exception of ServletException.`

Leave Websphere versions yet...